### DISCUSSION TITLE
Onnx model implementation in javscript

### DISCUSSION THREAD

**Discussion by shrestha-pranita** [snakers4/silero-vad#239] [Q&A]
Hello,

I have been trying to run the onnx model in javascript. I just wanted to ask if the format of the input is:
const inputs = new ort.Tensor("float32", new Float32Array([0.00064118, -0.0010624  -0.00098317, -0.00070163]),[1, 4]);

While running the onnx code in google colab, it seems the tensor is converted into numpy and then passed to the onnx session.

I am getting this error:

2022-09-21 16:43:14.565300 [E:onnxruntime:, sequential_executor.cc:368 Execute] Non-zero status code returned while running Conv node. Name:'Conv_81' Status Message: Invalid input shape: {1}

Thank you for your help.

**Comment by snakers4** [comment]
Hi,

Hope this thread helps - https://github.com/snakers4/silero-vad/discussions/216#discussioncomment-3412370
